README

```diff
@@ -1,3 +1,4 @@
 README
 create new branch feature-D
 feature-D
+feature-E
```

Comment:
Why add this feature-E not feature-F ?

Reply:
> Why add this feature-E not feature-F ?

Because this feature is special and we think this is the best way to solve the customer's problem.